cli-feed-v4.json

```diff
@@ -27172,7 +27172,7 @@
               "displayVersion": "v4",
               "targetFramework": ".NET 7",
               "description": "Isolated",
-              "endOfLifeDate": "2024-05-10T00:00:00Z"
+              "endOfLifeDate": "2024-05-11T00:00:00Z"
```

Comment:
If we're updating this here, .NET 7 end-of-support was May 14th per https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

It would also be great to get .NET 8 updated, as we seem to have the wrong date for it, too. It's correct for in-proc, but isolated has it 2 days late.